### PR TITLE
Graph chart

### PR DESCRIPTION
**Pull request by daiterranova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -559,4 +559,172 @@ const formatData = (data) => {
 ``` 
 The previous function mapping through the `data.t` property (corresponding to the timestamp of the stock) takes every timestamp and multiplies it for 1000 (because the API receives the time on miliseconds), and then assigns to the `x` property on the returned object. Overmore, the `y` property on the same object, receives as a value the `data.c` (corresponding to the close price) accordingly to the index that is being mapped.
 
-So, we create an object that matches the timestamp with the closest price of the stock requested, and within this format, we set the state variable `chartData` consider the period of time is pretend to show (a day, a week or a year).
\ No newline at end of file
+So, we create an object that matches the timestamp with the closest price of the stock requested, and within this format, we set the state variable `chartData` consider the period of time is pretend to show (a day, a week or a year).
```
Comment (reynico) [inline README.md]:
```suggestion
So, we create an object that matches the timestamp with the closest price of the stock requested, and within this format, we set the state variable `chartData` to consider the period of time it pretends to show (a day, a week or a year).
```

File: README.md [hunk 2]
```diff
@@ -559,4 +559,172 @@ const formatData = (data) => {
...
+
+### Creating chart
+
+We'll been using the [Apex Charts Library](https://apexcharts.com/) for showing the graph of the selected stock, implementing the integration for [React Charts](https://apexcharts.com/docs/react-charts/).
```
Comment (reynico) [inline README.md]:
```suggestion
We'll be using the [Apex Charts Library](https://apexcharts.com/) for showing the graph of the selected stock, implementing the integration for [React Charts](https://apexcharts.com/docs/react-charts/).
```

File: README.md [hunk 3]
```diff
@@ -559,4 +559,172 @@ const formatData = (data) => {
...
+            }
+        },
+        tooltip: {
+            x: {
+                format: "MMM dd HH:MM"
+            }
+        }
+    }
+```    
+Note: we'll configurate color to show the corresponding color depending on whether the stock goes up or down. For now we leave the line of code commented.
```
Comment (reynico) [inline README.md]:
```suggestion
Note: we'll configure color to show the corresponding color depending on whether the stock goes up or down. For now we leave the line of code commented.
```

File: README.md [hunk 4]
```diff
@@ -559,4 +559,172 @@ const formatData = (data) => {
...
+
+For additional information, you can go to the docs for options in the [apexcharts's web page](https://apexcharts.com/docs/options/).
```
Comment (reynico) [inline README.md]:
```suggestion
For additional information, you can go to the docs for options on the [apexcharts' web page](https://apexcharts.com/docs/options/).
```

File: README.md [hunk 5]
```diff
@@ -559,4 +559,172 @@ const formatData = (data) => {
...
+
+Return the `Chart` component and pass it the **options** property with the value of **options** object.
+
+Now we'll create the buttons so the user can choose between different periods of historical data's stock.
```
Comment (reynico) [inline README.md]:
```suggestion
Now we'll create the buttons so the user can choose between different periods of historical data stock.
```

File: README.md [hunk 6]
```diff
@@ -559,4 +559,172 @@ const formatData = (data) => {
...
+And pass it to every button:
+
+```
+ <div>
+            <button className={renderButtonSelect("24h")} onClick={() => { setDateFormat("24h") }}>24 hs</button>
+            <button className={renderButtonSelect("7d")} onClick={() => { setDateFormat("7D") }}>7 days</button>
+            <button className={renderButtonSelect("1y")} onClick={() => { setDateFormat("1Y") }}>1 Year</button>
+```
+
+`renderButtonSelect` asigns to all the buttons the bootstrap's classes `btn m-1` and, according to which button is active selected (what is defined by the value of dateFormat("24h, 7d or 1y")), add the class `btn-primary` or `btn-outline-primary`.
```
Comment (reynico) [inline README.md]:
```suggestion
`renderButtonSelect` assigns to all the buttons the bootstrap's classes `btn m-1` and, according to which button is active selected (what is defined by the value of dateFormat("24h, 7d or 1y")), add the class `btn-primary` or `btn-outline-primary`.
```

File: README.md [hunk 7]
```diff
@@ -559,4 +559,172 @@ const formatData = (data) => {
...
+```
+determineTimeFormat()[determineTimeFormat().length - 1].y - determineTimeFormat()[0].y
+```
+- With a ternary operator we evaluate if the result of the statement is greater than 0 (goes up), return **green color**; else, return **red color** 
+
+```
+    const color = determineTimeFormat()[determineTimeFormat().length - 1].y - determineTimeFormat()[0].y > 0 ? "#00FF00" : "#FF0000"
+```
+
+Now, we can uncommented the line inside of the **options** object:
```
Comment (reynico) [inline README.md]:
```suggestion
Now, we can uncomment the line inside of the **options** object:
```

File: src/App.css [hunk 8]
```diff
@@ -0,0 +1,7 @@
+.delete-button{
```
Comment (reynico) [inline src/App.css]: I'd suggest configuring the IDE linter/formatter, see that there's no space between `button` and `{`

**Review by reynico — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (daiterranova) [reply README.md]: changed

File: README.md (hunk 4 above)
Comment (daiterranova) [reply README.md]: everyday you can learn sth new

File: README.md (hunk 6 above)
Comment (daiterranova) [reply README.md]: ups

File: src/App.css (hunk 8 above)
Comment (daiterranova) [reply src/App.css]: Thanks for the suggestion. I'd already enable prettier to automate those things.

File: README.md (hunk 7 above)
Comment (daiterranova) [reply README.md]: amended

File: README.md (hunk 3 above)
Comment (daiterranova) [reply README.md]: changed

File: README.md (hunk 5 above)
Comment (daiterranova) [reply README.md]: done

**Review by reynico — APPROVED** [review]
lgtm!

**Pull request merged**